code/datums/components/bloody_spreader.dm

```diff
@@ -0,0 +1,45 @@
+/datum/component/bloody_spreader
+	dupe_mode = COMPONENT_DUPE_UNIQUE_PASSARGS
+	// How many bloodening instances are left. Deleted on zero.
+	var/blood_left
+	// We will spread this blood DNA to targets!
+	var/list/blood_dna
+	// For the love of god keep this null
+	var/list/diseases
```

Comment:
What's the point of this var if it has to stay null? That's not a nice comment (also it should be `///` for documentation purposes iirc)